components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -936,7 +936,7 @@ private DigestAuthConstants() {
     public static final String TENANTCOUNT_CACHEKEY = "apiProviderCacheKey";
 
     public static final String API_STORE_GROUP_EXTRACTOR_IMPLEMENTATION = API_STORE + "GroupingExtractor";
-
+    public static final String ENABLE_MULTIPLE_GROUPID = API_STORE +"EnableMultipleGroupId";
```

Comment:
The ideal name should be like "Enable Application Sharing based on Multiple groups IDs" so I choose to go with EnableMultipleGroupId since it was a long name. If there is shorter name please suggest. happy to change :)

Reply:
Removed configuration and used a default method to in LoginPostExecutor interface